pyiron_base/project/generic.py

```diff
@@ -245,7 +243,7 @@ def create_from_job(self, job_old, new_job_name):
         """
         job_id = self.get_job_id(new_job_name)
         if job_id is not None:
-            s.logger.info("create_from_job has already job_id {}!".format(job_id))
+            state.logger.info("create_from_job has already job_id {}!".format(job_id))
```

Comment:
Made the info more informative.
```suggestion
            state.logger.info(f"create_from_job: {new_job_name} has already job_id {job_id}!")
```